src/test/java/github/grace000/erversay/ResponseBuilderTest.java

```diff
@@ -0,0 +1,29 @@
+package github.grace000.erversay;
+
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class ResponseBuilderTest {
+
+    @Test
+    public void itHasADefaultCodeOf200() {
```

Comment:
These three tests are all identical. You're not really asserting individually here that it has a default code of 200. You're asserting that the response returned is `"HTTP/1.1 200 OK\r\nContent-Length: 0\r\n\r\n "`.

If you really want to assert that the response has 200, it would make more sense to interrogate that  `responseBuilder.code` equals 200. Since `code` is private, maybe that means a public getter `getStatusCode` method. Alternatively, maybe just collapse all three tests in this module into one.

+1 to what Brian says about the `Content-Length` response header; that would apply to all three tests as well.